ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusMessageEmojiReactions.qml

```diff
@@ -65,120 +69,102 @@ Item {
+                    if (reactionDelegate.hovered) {
+                        return Theme.palette.statusMessage.emojiReactionBackgroundHovered
                     }
+                    return model.didIReactWithThisEmoji ? Theme.palette.primaryColor2 : Theme.palette.statusMessage.emojiReactionBackground
+                }
+                border.width: model.didIReactWithThisEmoji || reactionDelegate.hovered ? 1 : 0
+                border.color: reactionDelegate.hovered ? Theme.palette.statusMessage.emojiReactionBorderHovered : Theme.palette.primaryColor1
+            }
 
+            contentItem: Row {
+                spacing: 4
+
+                StatusBaseText {
+                    objectName: "emojiReaction"
+                    font.pixelSize: Theme.fontSize17
+                    anchors.verticalCenter: parent.verticalCenter
+                    text: model.emoji
                 }
 
-                StatusToolTip {
-                    visible: reactionDelegate.hovered
-                    maxWidth: 400
-                    text: d.showReactionAuthors(model.jsonArrayOfUsersReactedWithThisEmoji, model.emoji)
+                StatusBaseText {
+                    anchors.verticalCenter: parent.verticalCenter
+                    text: model.numberOfReactions
+                    font.pixelSize: Theme.fontSize14
                 }
+            }
 
-                StatusMouseArea {
-                    id: mouseArea
-                    anchors.fill: parent
-                    hoverEnabled: true
-                    cursorShape: Qt.PointingHandCursor
-                    onEntered: {
-                        root.hoverChanged(true)
-                    }
-                    onExited: {
-                        root.hoverChanged(false)
-                    }
-                    onClicked: {
-                        root.toggleReaction(model.emoji)
-                    }
+            StatusToolTip {
+                visible: reactionDelegate.hovered
+                maxWidth: 400
+                text: d.showReactionAuthors(model.jsonArrayOfUsersReactedWithThisEmoji, model.emoji) || ""
+            }
+
+            StatusMouseArea {
+                id: mouseArea
+                anchors.fill: parent
+                hoverEnabled: true
+                cursorShape: Qt.PointingHandCursor
+                onEntered: {
+                    root.hoverChanged(true)
+                }
+                onExited: {
+                    root.hoverChanged(false)
+                }
+                onClicked: {
+                    root.toggleReaction(model.emoji)
                 }
             }
         }
+    }
 
-        Item {
-            width: addEmojiButton.width + addEmojiButton.anchors.leftMargin // there is more margin between the button and the emojis than between each emoji
-            height: addEmojiButton.height
-            visible: root.enabled
-
-            StatusIcon {
-                id: addEmojiButton
+    Item {
```

Comment:
This whole thing could be a small StatusButton with `enabled: !addEmojiButton.isHovered`